core/domain/user_services.py

```diff
@@ -775,16 +775,69 @@ def get_user_contribution_rights(user_id):
         UserContributionRights. The UserContributionRights domain object for the
         corresponding user.
     """
-    user_model = (
-        user_models.UserContributionRightsModel.get_by_id(user_id))
-    if user_model is not None:
-        return user_domain.UserContributionRights(
-            user_id,
-            user_model.can_review_translation_for_language_codes,
-            user_model.can_review_voiceover_for_language_codes,
-            user_model.can_review_questions)
-    else:
-        return user_domain.UserContributionRights(user_id, [], [], False)
+    return get_users_contribution_rights([user_id])[0]
+
+
+def get_users_contribution_rights(user_ids):
+    """Returns the UserContributionRights domain object for each user_id in
+    user_ids.
+
+    Args:
+        user_ids: list(str). A list of user ids.
+
+    Returns:
+        list(UserContributionRights). A list containing the
+        UserContributionRights domain object for each user.
+    """
+    user_contribution_rights_models = (
+        user_models.UserContributionRightsModel.get_multi(user_ids)
+    )
+
+    users_contribution_rights = []
+    for index, user_contribution_rights_model in enumerate(
+            user_contribution_rights_models):
+        if user_contribution_rights_model is not None:
+            users_contribution_rights.append(user_domain.UserContributionRights(
+                user_ids[index],
+                (
+                    user_contribution_rights_model
+                    .can_review_translation_for_language_codes
+                ),
+                (
+                    user_contribution_rights_model
+                    .can_review_voiceover_for_language_codes
+                ),
+                user_contribution_rights_model.can_review_questions))
+        else:
+            users_contribution_rights.append(
+                user_domain.UserContributionRights(
+                    user_ids[index], [], [], False))
+
+    return users_contribution_rights
+
+
+def get_reviewer_user_ids_to_notify():
+    """Gets a list of the reviewer user_ids who want to be notified of
+    Contributor Dashboard reviewer updates.
+
+    Returns:
+        list(str). A list of reviewer user_ids who want to be notified of
+        Contributor Dashboard reviewer updates.
+    """
+    # Get the user ids of the Contributor Dashboard reviewers.
+    users_contribution_rights = get_all_contribution_reviewers()
```

Comment:
get_all_contribution_reviewers() sounds like it's returning a list of objects representing reviewers, rather than contribution rights. Perhaps rename?

Or perhaps the name "users_contribution_rights" is wrong?

Reply:
The UserContributionRights domain object encompasses the reviewer's user_id and their reviewing permissions so in a way it is representing a reviewer? I can't think of any other information that would be included for a reviewer off the top of my head. I think whoever named this model named it UserContributionRights because the object is mostly a reviewers permissions. I agree that this wording is sightly confusing...I changed get_all_contribution_reviewers to get_all_reviewers_contribution_rights() - does that help? On a slightly separate note, I think user_reviewing_rights is more clear but maybe this model will be expanded on in the future to not include just reviewing permissions.

Reply:
I think this sounds good for now, thanks.

Reply:
Sounds good Sean :)